docs/src/main/tut/datatypes/functionk.md

```diff
@@ -121,3 +121,25 @@ type ErrorOr[A] = Either[String, A]
 val errorOrFirst: FunctionK[List, ErrorOr] =
   λ[FunctionK[List, ErrorOr]](_.headOption.toRight("ERROR: the list was empty!"))
 ```
+
+## Natural Transformation
+
+In category theory, a [Natural Transformation](https://en.wikipedia.org/wiki/Natural_transformation) provides a morphism between Functors while preserving the internal structures. It's one of the most fundamental notions of category theory.
```

Comment:
> while preserving the internal structures

I think _the internal structure_ is slightly better. It refers to the composition of morphisms (similarly to what a functor does)